main_exercise17.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+from exercise17 import calculate_male_bac, calculate_female_bac
+
+def main():
+    BAC_LEGAL_LIMIT = 0.08
+
+    gender = input("Enter your gender: male or female. ")
+    alcohol_amount = int(input("Enter the number of ounces of alcohol you've consumed: "))
+    weight = int(input("Enter your weight: "))
+    MESSAGE = print("Enter the number of hours since your last drink.")
+    hours_since_last_drink = int(input("If it's been less than 1 hour, type \'1\'. "))
+
+#calling functions
+    if gender == 'male':
+        male_bac_calcuation = calculate_male_bac(alcohol_amount, weight, hours_since_last_drink)
+    else:
+        female_bac_calculation = calculate_female_bac(alcohol_amount, weight, hours_since_last_drink)
+
+# coordinating output
+    if gender == 'male':
+        if male_bac_calcuation >= BAC_LEGAL_LIMIT:
+            rounded_bac_male = "{:.2f}".format(male_bac_calcuation)
```

Comment:
You need to move line 29 above line 28. This variable will only be defined it this first condition is met. It probably errors out on line 32 because it's not defined.